### DISCUSSION TITLE
CoApp: please add support for Waterfox browser

### DISCUSSION THREAD

**Discussion by Enkidu70** [aclap-dev/video-downloadhelper#70] [Help and Q&A]
### Your question

Please add support of www.waterfox.net (an enhanced and popular firefox derivate) into CoApp.

Tnx in advance.

### Addon & CoApp version and details

Latest

**Comment by jelabarre59** [comment]
As a workaround until additional browsers get added to the installer, how about the manual steps we could use to copy the settings from the firefox/mozilla profile?  Is it enough to copy ~/.mozilla/native-messaging-hosts/net.downloadhelper.coapp.json to ~/.waterfox/native-messaging-hosts/ ?
